packages/redux/src/raffles/selectors.ts

```diff
@@ -0,0 +1,89 @@
+import { getEntities, getEntityById } from '../entities/selectors';
+import {
+  getError as getErrorFromReducer,
+  getIsLoading as getIsLoadingFromReducer,
+} from './reducer';
+import type {
+  Raffle,
+  RaffleParticipation,
+} from '@farfetch/blackout-client/src';
+import type { RafflesState } from './types';
+import type { StoreState } from '../types/storeState.types';
+
+/**
+ * Returns all raffles from state.
+ *
+ * @param state - Application state.
+ *
+ * @returns Object with key values pairs representing raffleId and raffle properties.
+ */
+export const getRaffles = (state: StoreState) => getEntities(state, 'raffles');
```

Comment:
Question: Would it be useful to the consumer to receive the entity denormalised from this selector (and from the selector below `getRaffle`)? If yes, then we should return the entity denormalised here (be careful with performance though, you might need to use `reselect` for that. Check Wishlist selectors for examples on how to do that).

Reply:
pls double check